internal/slicer/slicer.go

```diff
@@ -30,6 +30,19 @@ func Run(options *RunOptions) (*Report, error) {
 	pathInfos := make(map[string]setup.PathInfo)
 	report := NewReport(options.TargetDir)
 
+	// TODO why not use knownPaths instead of having another list in report. It is because
+	// they have not the same use (they are similar by ""chance""). knownPaths is a list of
+	// all the paths that the Starlark script can touch including parent directories. For
+	// the report paths we only want the paths that were asked for explicitly in the slice
+	// definition. It happens now that both are similar but they could very well diverge more
+	// in the future.
+
+	// installedPaths contains all the paths that were extracted because they
+	// matched a path or a glob in the slice definition.
+	installedPaths := make(map[string]bool)
```

Comment:
Makes sense. There might be some nitpicks on the names though. But I don't have better suggestions. Only `explicitPaths` come to mind.